support/Pipelines/Bbh/Ringdown.py

```diff
@@ -144,6 +158,15 @@ def start_ringdown(
     default=5,
     show_default=True,
 )
+@click.option(
+    "--pipeline-dir",
+    "-d",
+    type=click.Path(
+        writable=True,
+        path_type=Path,
+    ),
+    help="Directory where steps in the pipeline are created.",
+)
 @scheduler_options
 def start_ringdown_command(**kwargs):
     _rich_traceback_guard = True  # Hide traceback until here
```

Comment:
[Random line] Is it easy to have some tests that all the pipeline directory stuff works, even if it's just `--no-submit` and checking some input files?

Reply:
I couldn't come up with a good test of the full pipeline because the steps actually need data from the previous steps. Of course we can and should add integration/regression tests that run the pipeline on a cluster.